README.md

```diff
@@ -1,26 +1,47 @@
-# Vagrant Developer Environment
+# Vagrant Developer VM
+
+## Tools Included
 
-## Tools included
 * Git
 * nvm
-* Atom Text Editor
-* IntelliJ Ultimate
-* Maven
-* Gradle
-* Open JDK
-* oh-my-zsh
-* Firefox
 * nano
+* oh-my-zsh
+* Open JDK
 * Python 3
+* Maven
+* Gradle
 * Docker
 * docker-compose
+* Atom Text Editor
+* Firefox
+* IntelliJ Ultimate
```

Comment:
Hab versucht eine etwas logische Ordnung in die Tools zu bringen.
Von lo-level über Sprachen bis zu mehr hi-level.